ci/code_checks.sh

```diff
@@ -83,6 +83,7 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
         pandas.Series.backfill \
         pandas.Series.ffill \
         pandas.Series.pad \
+        pandas.Series.dt.end_time \
```

Comment:
thanks for updating - looks like the check passes even with this in, are we sure we need it?

Reply:
I'll check

Reply:
looks like `Series.dt.end_time` inherits the docstrings from `Period.end_time`

![image](https://github.com/pandas-dev/pandas/assets/33491632/b90ad54e-96b1-4754-9619-a01342b0fb6b)


rather than doing complicated 4D code gymnastics to share docstrings, shall we just put the examples together? a bit like you did for some other examples, like
```
For period:
>>> [current example]

For Series:
>>> [example for Series.dt.end_time]
```

Reply:
Thanks - I saw this after I arrived to the same conclusion (but slower 🫤)